libs/api/database/tsconfig.lib.json

```diff
@@ -5,7 +5,8 @@
         "outDir": "../../../dist/out-tsc",
         "declaration": true,
         "types": ["node"],
-        "target": "es6"
+        "target": "es6",
+        "typeRoots": ["../../../node_modules/@types", "src/customTypes"]
```

Comment:
Now that we have some custom types, we need to tell the compiler where to find them. Since we do that, we have to tell it where to find _all_ of them, so the node_modules types go in here too.

We could import the `.d.ts` files manually, but who wants to do that everywhere?